lib/source-map-cache.js

```diff
@@ -20,16 +20,31 @@ SourceMapCache.prototype.applySourceMaps = function (coverage) {
   var mappedCoverage = _.cloneDeep(coverage)
 
   Object.keys(coverage).forEach(function (key) {
-    if (_this.cache[key]) {
-      _this._rewriteStatements(mappedCoverage[key], _this.cache[key])
-      _this._rewriteFunctions(mappedCoverage[key], _this.cache[key])
-      _this._rewriteBranches(mappedCoverage[key], _this.cache[key])
+    var sourceMap = _this.cache[key]
+    if (!sourceMap) {
```

Comment:
I encounter an issue with _rewritePath. The original path `src/lib/app.ls` become `src/lib/src/lib/app.ls` after _rewritePath.
And when I run `nyc report --reporter=lcov` in terminal, it emits an error: `Error: ENOENT: no such file or directory, open './src/lib/src/lib/app.ls'`